docs/architecture/adr-034-change-pubkey.md

```diff
@@ -0,0 +1,63 @@
+# ADR 034: Change PubKey
+
+## Changelog
+
+- 30-09-2020: Initial Draft
+
+## Status
+
+Proposed
+
```

Comment:
We require a summary section. Please have a look at the [template](https://github.com/cosmos/cosmos-sdk/blob/master/docs/architecture/adr-template.md)